content/src/service/ServiceImpl.ts

```diff
@@ -172,8 +177,7 @@ export class ServiceImpl implements MetaverseContentService, ClusterDeploymentsS
                     await this.storeEntityContent(hashes, alreadyStoredContent)
 
                     // Since we are still reporting the history size, add one to it
-                    await this.historyManager.reportDeployment(transaction.deployments)
-
+                    this.historySize++
```

Comment:
Let's move this outside the transaction, since it could potentially fail for some reason and we would still be counting  the new deployment. We can add it inside the same `if` that calls the listeners